packages/common/src/types/command/command.types.ts

```diff
@@ -1,14 +1,19 @@
-import type { ActionCommand } from "./ActionCommand";
-import type { CommandV5 } from "./CommandV5.types";
+import { CommandV6 } from "./CommandV6.types";
 import type { CommandV0, CommandV1 } from "./legacy/CommandV0V1.types";
 import type { CommandV2 } from "./legacy/CommandV2.types";
 import type { CommandV3 } from "./legacy/CommandV3.types";
 import type { CommandV4 } from "./legacy/CommandV4.types";
+import type { CommandV5 } from "./legacy/CommandV5.types";
 
-export type CommandComplete = Required<Omit<CommandLatest, "spokenForm">> &
-  Pick<CommandLatest, "spokenForm"> & { action: Required<ActionCommand> };
+// TODO:
+// export type CommandComplete = Required<Omit<CommandLatest, "spokenForm">> &
+//   Pick<CommandLatest, "spokenForm"> & {
+//     action: Required<PartialActionDescriptor>;
+//   };
```

Comment:
We can't make all fields in the action descriptor required since we have optional parameters where the default value is set in the action.